### PR TITLE
Derive count of api servers from etcd config

### PR DESCRIPTION
**Pull request by lewismarshall**
Will use the environment variable / etcd flag to derive the correct number of api servers. API servers will go down unless they know of each other.

### CODE REVIEW

File: pkg/kmm/cmd/etcdCerts.go [hunk 1]
```diff
@@ -48,13 +48,25 @@ func init() {
...
 }
 
+func GetEtcdHostNames(cmd *cobra.Command, minimalDefaultHosts []string) ([]string, error) {
+	var err error
+	etcdClusterHostnames := strings.Split(cmd.Flag("etcd-cluster-hostnames").Value.String(), ",")
+	if len(etcdClusterHostnames) - 1 == 0 {
+		var etcdClusterUrls string
+		if etcdClusterUrls, err = GetUrlsFromInitialClusterString(os.Getenv("ETCD_INITIAL_CLUSTER")); err != nil {
+			return []string{}, err
+		} else {
```
Comment (vaijab) [inline pkg/kmm/cmd/etcdCerts.go]: else is not needed here.

Comment (lewismarshall) [inline pkg/kmm/cmd/etcdCerts.go]: True - I'll push amended...

File: pkg/kubeadm/manifests.go [hunk 2]
```diff
@@ -6,14 +6,12 @@ import (
...
+	// Get config into kubeadm format
 	var cfg *kubeadmapi.MasterConfiguration
 	if cfg, err = GetKubeadmCfg(kubeadmCfg); err != nil {
 		return err
 	}
-	if cfg == nil {
-		return nil
-	}
-	if master.WriteStaticPodManifests(cfg) ; err != nil {
+	if master.WriteStaticPodManifests(cfg, kubeadmCfg.MasterCount) ; err != nil {
```
Comment (vaijab) [inline pkg/kubeadm/manifests.go]: Just return `master.WriteStaticPodManifests()` as it returns an error anyway.

Comment (lewismarshall) [inline pkg/kubeadm/manifests.go]: 👍 pushing change...

**Pull request merged**